.github/workflows/nightly.yml

```diff
@@ -4,6 +4,10 @@ on:
     - cron: '0 1 * * *'
   # can be run manually on https://github.com/rhinstaller/anaconda-webui/actions
   workflow_dispatch:
+    inputs:
+      compose_id:
+        description: 'Compose ID to test'
+        required: true
```

Comment:
The code supports not specifying it (then it just tests latest rawhide), so this feels unnecessary/too strong?

Reply:
True, reposting.